pkg/clients/edgeconnect/environment_settings.go

```diff
@@ -0,0 +1,159 @@
+package edgeconnect
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/clients/utils"
+	"github.com/pkg/errors"
+)
+
+const (
+	KubernetesConnectionSchemaID = "app:dynatrace.kubernetes.connector:connection"
+	KubernetesConnectionVersion  = "0.1.1"
+	KubernetesConnectionScope    = "environment"
+)
+
+type EnvironmentSetting struct {
+	ObjectId      *string                 `json:"objectId"`
+	SchemaId      string                  `json:"schemaId"`
+	SchemaVersion string                  `json:"schemaVersion"`
+	Scope         string                  `json:"scope"`
+	Value         EnvironmentSettingValue `json:"value"`
+}
```

Comment:
more of a note:

looks awfully familiar to https://github.com/Dynatrace/dynatrace-operator/blob/8b5ac38fd58335674e93353747ea430abcba36a5/pkg/clients/dynatrace/kubernetes_settings.go#L33-L39

we should look into working with the SettingsApi in a more general way if possible:
- the `kubernetes_settings.go` is rather old, so I wouldn't base it necessarily on it, but would be nice to have a more general implementation for a more general use API (if I understand the SettingsApi correctly)

Reply:
Let's do this in a follow up.

Reply:
okey so @albertogdd please create a ticket before closing the thread, to not forget.

Reply:
Fine for me :+1: Will do